src/apiCalls.js

```diff
@@ -2,10 +2,21 @@ const getData = (dataSource) => {
     return fetch(`http://localhost:3001/api/v1/${dataSource}`)
         .then(response => {
             if(!response.ok) {
-                throw new Error(`${response.status}`)
+                throw new Error("Sorry, we were unable to retreive your account information. Please try again.")
             }
             return response.json()
         })
 }
 
-export default getData
\ No newline at end of file
+const postData = (newBooking) => {
+    console.log("Boo")
+    return fetch('http://localhost:3001/api/v1/bookings', newBooking)
+        .then(response => {
+            if(!response.ok) {
+                throw new Error("Sorry, your booking request did not go through. Please try again.")
+            }
+            return response.json()
+        })
+}
```

Comment:
I would add a .catch here, make sure to inform the user of the error in the .catch. It will catch all errors from the fetch, so I would make it generic, something like "Oops something went wrong"